From: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Date: Fri, 5 Sep 2025 17:04:24 -0600
Subject: Remove google custom engine queries script

Origin: other
Forwarded: not-needed
Last-Update: 2016-09-29

Remove javascript for the custom google engine queries, in the header
and in the footer of each html documentation page, to avoid potential
privacy breach.
---
 doc/html/header.txt | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/doc/html/header.txt b/doc/html/header.txt
index 59eb454..7fd276f 100644
--- a/doc/html/header.txt
+++ b/doc/html/header.txt
@@ -5,7 +5,6 @@
 	<meta name="viewport" content="width=device-width">
 
 	<!--#pagetitle-->
-	<!--#canonical-->
 
 	<link rel="shortcut icon" href="favicon.ico">
 
@@ -15,17 +14,6 @@
 	<link rel="stylesheet" type="text/css" href="slurm.css">
 
 	<script src="jquery.min.js"></script>
-	<script type="text/javascript">
-	jQuery(document).ready(function() {
-		jQuery('.menu-trigger').bind('click touchstart', function() {
-			jQuery(this).find('.menu-trigger__lines').toggleClass('menu-trigger__lines--closed');
-			jQuery(this).parents('.site-header').find('.site-nav').toggleClass('site-nav--active');
-
-			return false;
-		});
-	});
-	</script>
-	<script async src="https://cse.google.com/cse.js?cx=011890816164765777536:jvrtxrd3f0w"></script>
 </head>
 
 <body>
